chapters/overloaded.tex

```diff
@@ -276,15 +282,19 @@ \section{Overloaded Binary Operations}\doublelabel{overloaded-binary-operations}
     type. If the operator \lstinline!'0'! is not defined for that class it is an
     error if the inner dimension is zero.
   \end{enumerate}
-{[}\emph{For array multiplication it is assumed that the scalar elements
+
+\begin{nonnormative}
+For array multiplication it is assumed that the scalar elements
 form a non-commutative ring that does not necessarily have a
-multiplicative identity.}{]}
+multiplicative identity.
+\end{nonnormative}
+
 \item
   Otherwise the expression is erroneous.
 \end{enumerate}
 
 For an element-wise operator, a .op b, items 1, 4, and 5 are used; e.g.
-the operator .+ will always be defined in terms of \lstinline!'+'.!
+the operator \lstinline!.+! will always be defined in terms of \lstinline!'+'.!
```

Comment:
The end of the sentence was broken before as well, it should be in terms of \lstinline!'+'!. 
(that dot is just an end of sentence.)

Reply:
Fixed.

After having looked at lots of the source code lately, I actually get a bit concerned about missing or incorrect punctuation in general.  It makes me think that we should consider paying someone for proof reading the entire text…